site/src/util/mdx-utils.tsx

```diff
@@ -31,6 +33,7 @@ const isParent = (node: Node): node is Parent => "children" in node;
 type Heading = {
   type: "heading";
   depth: number;
+  anchor?: string;
```

Comment:
You no longer need this

Reply:
Removed in 90b9a7c2d74f6b9a3b894ae5913e9a3585295dba